homeassistant/components/hassio/http.py

```diff
@@ -89,9 +90,10 @@ async def _command_proxy(
             request._client_max_size = (  # pylint: disable=protected-access
                 MAX_UPLOAD_SIZE
             )
+            client_timeout = 300
```

Comment:
Why not disable timeout?

Reply:
We could, but then it would never timeout :shrug: 
worst case you upload 1GB, with 5min it will still timeout if your connection is slower than 29Mb/s.
This should only be done in a local network, I think that is a reasonable max value?

But if you feel we should have `None` (disabled) I can update it.

Reply:
So is this timeout only for the uploading, or also the processing?

Reply:
Only the upload, and only the first part of the upload (between the client (browser) and HA).
When that part is done, HA sends this to SU which will process it (save).

Reply:
Maybe 10min for slow SD?

Reply:
Writing to SD is not a part of this

Reply:
I guess 5 min is ok.